modules/apps/expando/expando-test/src/testIntegration/java/com/liferay/expando/service/test/ExpandoValueLocalServiceTest.java

```diff
@@ -169,6 +170,32 @@ public void testAddNondefaultLocalizedStringValue() throws Exception {
 		}
 	}
 
+	@Test
+	public void testAddSiteDefaultLocalizedStringValue() throws Exception {
+		Locale originalLocale = LocaleUtil.getSiteDefault();
+
+		LocaleThreadLocal.setSiteDefaultLocale(LocaleUtil.FRANCE);
+
+		ExpandoColumn column = ExpandoTestUtil.addColumn(
+			_expandoTable, "Test Column",
+			ExpandoColumnConstants.STRING_ARRAY_LOCALIZED);
+
+		ExpandoValue value = ExpandoTestUtil.addValue(
+			_expandoTable, column,
+			HashMapBuilder.put(
+				_frLocale, "TestFR"
+			).build());
+
```

Comment:
@crodriguezy, ExpandoTestUtil.addValue() will throw one exception. So the test can't be passed. Please see the error stack
```
com.liferay.expando.service.test.ExpandoValueLocalServiceTest > testAddSiteDefaultLocalizedStringValue FAILED
    java.lang.ClassCastException: java.lang.String cannot be cast to [Ljava.lang.String;
        at com.liferay.portlet.expando.model.impl.ExpandoValueImpl.setStringArrayMap(ExpandoValueImpl.java:525)
        at com.liferay.portlet.expando.service.impl.ExpandoValueLocalServiceImpl.addValue(ExpandoValueLocalServiceImpl.java:404)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:50)
        at com.liferay.portal.spring.transaction.TransactionInterceptor.invoke(TransactionInterceptor.java:69)
        at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:57)
        at com.liferay.portal.spring.aop.AopInvocationHandler.invoke(AopInvocationHandler.java:49)
        at com.sun.proxy.$Proxy173.addValue(Unknown Source)
        at com.liferay.portlet.expando.service.impl.ExpandoValueLocalServiceImpl.addValue(ExpandoValueLocalServiceImpl.java:584)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:50)
        at com.liferay.portal.spring.transaction.TransactionInterceptor.invoke(TransactionInterceptor.java:69)
        at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:57)
        at com.liferay.portal.spring.aop.AopInvocationHandler.invoke(AopInvocationHandler.java:49)
        at com.sun.proxy.$Proxy173.addValue(Unknown Source)
        at com.liferay.expando.kernel.service.ExpandoValueLocalServiceUtil.addValue(ExpandoValueLocalServiceUtil.java:225)
        at com.liferay.portlet.expando.util.test.ExpandoTestUtil.addValue(ExpandoTestUtil.java:65)
        at com.liferay.portlet.expando.util.test.ExpandoTestUtil.addValue(ExpandoTestUtil.java:87)
        at com.liferay.expando.service.test.ExpandoValueLocalServiceTest.testAddSiteDefaultLocalizedStringValue(ExpandoValueLocalServiceTest.java:183)
```